src/com/itacademy/eshop/product/Card.java

```diff
@@ -0,0 +1,61 @@
+package com.itacademy.eshop.product;
+
+import com.itacademy.eshop.shop.ShoppingCart;
+
+public abstract class Card extends Payment {
+
+    private String cardNumber;
+    private String cardHolder;
+    private String expirationDate;
+    private int cvvNumber;
+
+    public Card(String paymentMethod, double amount, String cardNumber, String cardHolder, String expirationDate, int cvvNumber) {
+        super(paymentMethod, amount);
+        this.cardNumber = cardNumber;
+        this.cardHolder = cardHolder;
+        this.expirationDate = expirationDate;
+        this.cvvNumber = cvvNumber;
+    }
+
+    public String getCardNumber() {
+        return cardNumber;
+    }
+
+    public void setCardNumber(String cardNumber) {
+        this.cardNumber = cardNumber;
+    }
+
+    public String getCardHolder() {
+        return cardHolder;
+    }
+
+    public void setCardHolder(String cardHolder) {
+        this.cardHolder = cardHolder;
+    }
+
+    public String getExpirationDate() {
+        return expirationDate;
+    }
+
+    public void setExpirationDate(String expirationDate) {
+        this.expirationDate = expirationDate;
+    }
+
+    public int getCvvNumber() {
+        return cvvNumber;
+    }
+
+    public void setCvvNumber(int cvvNumber) {
+        this.cvvNumber = cvvNumber;
+    }
+
+    @Override
+    public void processPayment(ShoppingCart shoppingCart) {
```

Comment:
since you are only using shoppingCart.getTotalPrice, it would make more sense to only pass that value instead of the entire shoppingCart object to this method